hw/xbox/xbox.c

```diff
@@ -253,7 +253,35 @@ void nv2a_init(PCIBus *bus, int devfn, MemoryRegion *ram);
 /* PC hardware initialisation */
 static void xbox_init(MachineState *machine)
 {
-    xbox_init_common(machine, default_eeprom, NULL, NULL);
+    char *filename;
```

Comment:
This works for Xbox machine but would have to be duplicated for Chihiro support. We should think of a common solution so both can support it via the same code.